packages/webapp/src/config.ts

```diff
@@ -123,3 +127,9 @@ export const jobKeys = {
 export const eosPrivateKeys = {
     gcJob: process.env.EOS_PRIVATE_KEY_GC_JOB || "",
 };
+
+export const zoom = {
+    clientKey: process.env.NEXT_PUBLIC_ZOOM_CLIENT_ID || "",
+    clientSecret: process.env.ZOOM_CLIENT_SECRET || "",
+    oauthRedirect: `http://localhost:3000/oauth/zoom`, // todo: make the domain dynamic
```

Comment:
Does your plugin catch "todo" lowercase? The one I use i believe require it to be "TODO" to be picked up. I just don't want this to be lost.

Reply:
changed to uppercase. I actually don't have a plugin. I do a glob search for `todo:`. But good to know, I will use that one! :D